tasks/scripts/packer-custom-image

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e -u
+
+image_name="u16k44"
+outputs=$PWD/image-name
+
+echo "$image_name" > outputs/image-name
+
+tar -xzvf linux-kernel-44/linux-kernel-v4.4*.tgz -C ./ci/deployments/legacy-kernel
+
+cd ci/deployments/legacy-kernel
+
+echo "$GCP_KEY" > gcp.json
+export GOOGLE_APPLICATION_CREDENTIALS=gcp.json
+
+packer build \
+  -var region="us-west1" \
+  -var machine_type="n1-standard-1" \
+  -var zone="us-west1-b" \
```

Comment:
What are the implications of the region and machine type? Does it need to match exactly with the size/location of the VM it's used for?

If so, does the terraform need to be updated? https://github.com/concourse/ci/blob/3200d4f2ab4fca3a3501d77852606957d3b76f2f/deployments/legacy-kernel/main.tf#L3-L9

Reply:
Interesting. The image that created in `us-west1` has a location `us`, from what I can see in gcp console. It seems out terraform smoke deployment that configured to `us-central1` can find it without problem. So I think as long as they are all located within US the regions/zones don't need to match.

Reply:
Cool, thanks. Also, is there a reason for going with `f1-micro`? Seems really small (1 vCPU, 0.6 GB memory)

Reply:
this `main.tf` file is only used for testing if the packer created image could be used by terraform to create a VM. So it is for the script in https://github.com/concourse/ci/blob/15bc8429c18d9e894dae2b2c884e105d75120998/deployments/legacy-kernel/README.md.